aodntools/timeseries_products/velocity_aggregated_timeseries.py

```diff
@@ -0,0 +1,373 @@
+
+    ## add atributes
+    with open(TEMPLATE_JSON) as json_file:
+        attribute_dictionary = json.load(json_file)
+    variable_attribute_dictionary = attribute_dictionary['_variables']
+    global_attribute_dictionary = attribute_dictionary['_global']
+
+    ## set variable attrs
+    for var in list(ds.variables):
+        ds[var].setncatts(variable_attribute_dictionary[var])
+
+    if download_url_prefix or opendap_url_prefix:
+        ds['source_file'].setncatts(TStools.source_file_attributes(download_url_prefix, opendap_url_prefix))
+
+    ## set global attrs
+    timeformat = '%Y-%m-%dT%H:%M:%SZ'
+    file_timeformat = '%Y%m%d'
+
+    time_start = nc4.num2date(np.min(TIME[:]), time_units, time_calendar).strftime(timeformat)
+    time_end = nc4.num2date(np.max(TIME[:]), time_units, time_calendar).strftime(timeformat)
+    time_start_filename = nc4.num2date(np.min(TIME[:]), time_units, time_calendar).strftime(file_timeformat)
+    time_end_filename = nc4.num2date(np.max(TIME[:]), time_units, time_calendar).strftime(file_timeformat)
+
+    contributor_name, contributor_email, contributor_role = TStools.get_contributors(files_to_agg=files_to_agg, input_dir=input_dir)
+    add_attribute = {
+                    'title':                    ("Long Timeseries Velocity Aggregated product: " + ', '.join(varlist) + " at " +
+                                                  site_code + " between " + time_start + " and " + time_end),
+                    'site_code':                site_code,
+                    'time_coverage_start':      time_start,
+                    'time_coverage_end':        time_end,
+                    'geospatial_vertical_min':  np.min(ds['DEPTH']),
+                    'geospatial_vertical_max':  np.max(ds['DEPTH']),
+                    'geospatial_lat_min':       np.min(ds['LATITUDE']),
+                    'geospatial_lat_max':       np.max(ds['LATITUDE']),
+                    'geospatial_lon_min':       np.min(ds['LONGITUDE']),
+                    'geospatial_lon_max':       np.max(ds['LONGITUDE']),
+                    'date_created':             datetime.utcnow().strftime(timeformat),
+                    'history':                  datetime.utcnow().strftime(timeformat) + ': Aggregated file created.',
+                    'keywords':                 ', '.join(varlist + ['AGGREGATED']),
+                    'rejected_files':           "\n".join(rejected_files),
+                    'contributor_name':        "; ".join(contributor_name),
+                    'contributor_email':       "; ".join(contributor_email),
+                    'contributor_role':        "; ".join(contributor_role),
+                    'generating_code_version':  __version__
+    }
+
+    ## add version
+    github_comment = ('\nThis file was created using https://github.com/aodn/python-aodntools/blob/'
+                      '{v}/aodntools/timeseries_products/aggregated_timeseries.py'.format(v=__version__)
```

Comment:
Forgot to update the file name! Actually, we could do this instead...
```suggestion
                      '{v}/aodntools/timeseries_products/{f}'.format(v=__version__, f=os.path.basename(__file__))
```